### PR TITLE
(build): release armv7 binary

### PR DESCRIPTION
**Pull request by allendema**
The armv7 binary will be build when using goreleaser with `release.yml`.
The produced output was previously tested on a rpi.

### CODE REVIEW

**Comment by aleksasiriski** [comment]
Hmm, I disabled x86 builds on purpose because we have some 64bit only code I believe. I can check and if it's possible to fix this code to support 32bit systems then we could make armv7 builds.

You say that it works for you or RPi?

**Comment by allendema** [comment]
Version [v0.29.1](https://github.com/hearchco/agent/releases/tag/v0.29.1) works on rpi 4. 
Used a goreleaser template with these values modified - and there were results returned:
```yaml
  - env:
      - CGO_ENABLED=0
    goos:
      - linux
    goarch:
      - arm
    goarm:
      - 7
```

**Comment by aleksasiriski** [comment]
Could you also add goreleaser config for building the armv7 docker image? I believe it uses the following Docker platform: `linux/arm/v7`

**Comment by aleksasiriski** [comment]
It seems with Go int64 works on 32bit systems, by using 2x32bit registers and simulating 64bit arithmethic which is slower (but in our use case it's just a few consts so no problem, although I need to check for one bug that could appear before merging this).

**Comment by allendema** [comment]
>>  config for building the armv7 docker image?

> Yeah, pushed the changes just now.

Edit:  Also added it to "image_templates".

**Review by aleksasiriski — APPROVED** [review]
LGTM, let's see if it works :)

**Comment by aleksasiriski** [comment]
Sadly, gonna have to wait for rate limits to cool down, never encountered this before: <https://github.com/hearchco/agent/actions/runs/11330271635/job/31507720326>

**Comment by aleksasiriski** [comment]
Should be released soon under v0.29.2

**Comment by aleksasiriski** [comment]
Thanks for your contribution and do let me know how v0.29.2 works out for you :)

**Comment by allendema** [comment]
Nothing.
Will give it a spin soon.  :)

**Comment by allendema** [comment]
Just tested it and it works!

**Comment by aleksasiriski** [comment]
Just a follow up: today I've added and released frontend docker image that supports arm/v7 in case you need it

**Pull request merged**
